aspnetcore/signalr/javascript-client.md

```diff
@@ -52,50 +51,51 @@ To prevent a malicious site from reading sensitive data from another site, [cros
 
 ## Call hub methods from client
 
-JavaScript clients call public methods on hubs by using `connection.invoke`. The `invoke` method accepts two arguments:
+JavaScript clients call public methods on hubs via the [invoke](/javascript/api/%40aspnet/signalr/hubconnection#invoke) method of the [HubConnection](/javascript/api/%40aspnet/signalr/hubconnection). The `invoke` method accepts two arguments:
 
 * The name of the hub method. In the following example, the method name on the hub is `SendMessage`.
 * Any arguments defined in the hub method. In the following example, the argument name is `message`.
 
-[!code-javascript[Call hub methods](javascript-client/sample/wwwroot/js/chat.js?range=24)]
+  [!code-javascript[Call hub methods](javascript-client/sample/wwwroot/js/chat.js?range=24)]
```

Comment:
nit: remove space?

Reply:
That was intentional to indent the code snippet. This change visually associates the snippet with the bullet point above it.